boards/parts/ui/glfw/config.cmake

```diff
@@ -3,19 +3,20 @@ include("${OTTO_EXTERNAL_DIR}/gl3w_gen.cmake")
 gl3w_gen("${OTTO_EXTERNAL_DIR}/")
 
 # GLFW
-find_package(glfw3)
-if (glfw3_FOUND)
-    message("Using system GLFW")
-    if(NOT APPLE) 
-        target_link_libraries(glfw INTERFACE dl)
-    endif()
-else() 
+#find_package(glfw3)
+#if (glfw3_FOUND)
+#    message("Using system GLFW")
+#    #pkg_search_module(GLFW REQUIRED glfw3)
+#    include_directories(${GLFW_INCLUDE_DIRS})
+#    target_link_libraries(otto ${GLFW_LIBRARIES})
+#
+#    #target_link_libraries(glfw INTERFACE dl)
+#else() 
```

Comment:
we should fix this instead of removing it

Reply:
Definitely, it was a hack not meant to appear here ^^